src/simple_commander/main.py

```diff
@@ -449,6 +449,7 @@ def set_invaders(self, count):
     def get_units(self):
         units = []
         if len(self.units):
+            #TODO: limit units to top 10
```

Comment:
But this method returns list of units,not just statistics. If we start returning the statistics in separate method like `get_units_stats` so there would be good place to add such limitation.

But anyway we can't add any kind of limitation because scoreboard that we have is adaptive. It means that new users will see scoreboard of active heroes + who started after them. New users won't see statistics (even high rates) of old games and heroes who died before their registration